torch/testing/_internal/common_optimizers.py

```diff
@@ -857,12 +892,28 @@ def optim_error_inputs_func_sparseadam(device, dtype):
         optim_inputs_func=optim_inputs_func_rmsprop,
         optim_error_inputs_func=optim_error_inputs_func_rmsprop,
         supported_impls=("foreach", "differentiable"),
+        skips=(
+            # See https://github.com/pytorch/pytorch/issues/115679
```

Comment:
nit: this one and the one below: move the issue into the skip message like you do above.

Reply:
oh wow i forgot i was in the middle of doing that